### DISCUSSION TITLE
404: NOT_FOUND - CSS and JS not loading

### DISCUSSION THREAD

**Discussion by andresklein** [vercel/vercel#11622] [Help]
### Question

Hi, I'm trying to upload my site to Vercel but the css and JS are not working. Now I added the /src folder and the html is not working either. before the /src folder the HTML was working, but not the CSS and JS.
Help! Thank you!!!
https://roll-kayakismo.vercel.app/
https://github.com/andresklein/rollKayakismo

**Comment by EuFontoura** [comment]
Hey, Andre! Did you try using relative paths in your CSS and JS links? For example, `<link rel="stylesheet" href="./src/css/style.css" />` instead of `<link rel="stylesheet" href="/src/css/style.css" />`.

The path starting with `./` indicates a path relative to the current location of the HTML document, while the path starting with a slash `/`indicates an absolute path from the root of the site.

**Reply by andresklein** [reply]
Hey mate, thanks for your answer.. yes, I tried but nothing.. I didn't find the solution..

**Comment by math-silva** [comment]
**Hi there!** The issue you're facing is related to how Vercel handles static assets. Here's the fix:

**Understanding the Problem**

Vercel expects your static files (HTML, CSS, images) to be either in your project's root directory or inside a special folder named `public`. Your current structure has the assets in a `src` folder, which Vercel doesn't serve directly for static sites.

**Solution: Use a `public` Folder**

1. **Create a `public` Folder:** In the root of your project (the same level as `index.html`), create a new folder named `public`.
2. **Move Assets:** Move your `images` and `css` folders into this new `public` folder.
3. **Update Paths:**  Change the paths in your `index.html` file to be relative to the `public` folder. For example:

   * Instead of `/src/css/style.css`, use `css/style.css`
   * Instead of `/src/images/logo.png`, use `images/logo.png`

**Example Project Structure**

Here's how your project should look:

```
- rollKayakismo/
  - public/
    - css/
      - style.css
    - images/
      - ...
  - index.html
  - vercel.json 
```

**Redeploy:** After making these changes, redeploy your site to Vercel.

**Let me know if this helps!** If it solves your issue, please consider marking this answer as accepted. It helps others who might have the same question. 😊
